docs/developers/README.md

```diff
@@ -48,9 +46,7 @@ expected in directory path `/workspace/output/resource_name`.
     type: storage
   ```
 
-  - Copy step includes resource being copied to PVC to path
-    `/pvc/task_name/resource_name` from `/workspace/random-space/` if resource
-    is declared in input resource like below example.
+  - If resource is declared both in input and output for task then copy step includes resource being copied to PVC to path `/pvc/task_name/resource_name` from `/workspace/random-space/` if input resource has custom target directory(`random-space`) declared like the following example.
```

Comment:
In the different examples here, my understanding is that the resource will be copied to the PVC in the path  `/pvc/task_name/resource_name` in all of the cases, if this is correct why do we need 3 examples?

Reply:
We need 2 examples to show that source path when copied to PVC can change depending on 
1. resource only in output
2. resource in input and output (no target path)
3. resource in input and output( with target path)